app/BUILD.bazel

```diff
@@ -8,14 +8,43 @@ load("@tools_android//tools/crashlytics:defs.bzl", "crashlytics_android_library"
 load("@tools_android//tools/googleservices:defs.bzl", "google_services_xml")
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
 
-# Source files for annotations library.
+'''
+Source files for the annotations library. The files inside the annotations library and the listeners
+library are dependencies for all other libraries in app module.
+
+Place your file here if:
+ - A view, view model, or layout file depends on it
+ - It is an annotation or helper file of some kind
+ - It is not considered a listener
+
+DO NOT place your file here if:
+ - It depends on any other file not included in this list
+ - Any files in the listeners library depends on it
+ - It uses Kotlin synthetic imports
+ - Only test files depend on it
```

Comment:
Suggest striking the 'do not place' unless you're concerned with people specifically doing that in certain cases. I'm hoping that the 'place your file here' implies sufficient exclusivity that, when compared with other lists & their instructions, it should be clear to developers where to put new files when they need to.

Reply:
Okay